internal/characters/kokomi/abil.go

```diff
@@ -214,10 +217,6 @@ func (c *char) Burst(p map[string]int) (int, int) {
 	if c.Core.Status.Duration("kokomiskill") > 0 {
 		// +1 to avoid same frame expiry issues with skill tick
 		c.Core.Status.AddStatus("kokomiskill", 12*60+1)
-		c.skillLastUsed = c.Core.F - 1
-		d := c.createSkillSnapshot()
-		// Tick intervals stay the same if duration is refreshed
-		c.AddTask(c.skillTickTask(d, c.Core.F), "kokomi-e-ticks", 120-(c.Core.F-c.skillLastTick))
```

Comment:
I don't think you can remove this since Kokomi's E gets re-snapshot when you cast her Q.

Reply:
Seems like the burst doesn't re-snapshot the skill, only extends the duration.

calc: https://i.imgur.com/ifTczDY.png
video: https://www.youtube.com/watch?v=vmdj37M2Dxw

Reply:
Sounds good, then we can go ahead and merge this. Make sure to submit something to the KQM website feedback too so the library gets fixed before someone else comes in and asks about it at some point down the line lol